cypress/e2e/smoke/transactionalEmails/accountClosure.cy.js

```diff
@@ -0,0 +1,17 @@
+describe('QATEST-162618 - Verify account_closure are triggered as transactional email', () => {
+  beforeEach(() => {
+    cy.c_createRealAccount()
+    cy.c_login()
+  })
+  it(`Verify account_closure are triggered as transactional emails on Mobile`, () => {
+    cy.c_visitResponsive('account/closing-account')
```

Comment:
Better to pass size here otherwise it takes default size from env file which might change in future.
```suggestion
    cy.c_visitResponsive('account/closing-account', 'small')
```